physionet-django/events/templates/events/event_detail.html

```diff
@@ -0,0 +1,88 @@
+{% extends 'base.html' %}
+
+{% load static %}
+
+{% block title %}Event - {{ event.title }}{% endblock %}
+
+{% block local_css %}
+<link rel="stylesheet" type="text/css" href="{% static 'project/css/project-home.css' %}">
+{% endblock %} 
+
+{% block content %}
+<div class="container">
+{% if event %}
+    <h2 class="form-signin-heading">{{ event.title }}</h2>
+    <div class="col-md-9 no-pd">
+        <br/>
+        <h4>Event Details</h4>
+        <hr>
+{#        <div class="row mb-1">#}
+{#            <div class="col-md-3">#}
+{#                Hosted by:#}
+{#            </div>#}
+{#            <div class="col-md-9">#}
+{#                {{ event.host.get_full_name }}#}
+{#            </div>#}
+{#        </div>#}
+        <div class="row mb-1">
+            <div class="col-md-3">
+                Category:
+            </div>
+            <div class="col-md-9">
+                {{ event.category }}
+            </div>
+        </div>
+        <div class="row mb-1">
+            <div class="col-md-3">
+                Created on:
+            </div>
+            <div class="col-md-9">
+                {{ event.added_datetime | date:"M. d, Y" }}
+            </div>
+        </div>
+
+        <div class="row mb-1">
+            <div class="col-md-3">
+                Start Date:
+            </div>
+            <div class="col-md-9">
+                {{ event.start_date | date:"M. d, Y"}}
+            </div>
+        </div>
+        <div class="row mb-1">
+            <div class="col-md-3">
+                End Date:
+            </div>
+            <div class="col-md-9">
+                {{ event.end_date | date:"M. d, Y" }}
+            </div>
+        </div>
+
+        <div class="row mb-1">
+            <div class="row col-md-3">
+                Description:
+            </div>
+            <div class="row col-md-12">
+                {{ event.description }}
+            </div>
+        </div>
+
+
+
+    </div>
+        <hr>
+
+
+    <br>
+    <p>You may request to join this event below.</p>
+
+    <strong><p>Requesting participation adds your details to a waiting list. You will receive a notification when your request has been processed.</p></strong>
+
+    <form  method="post">
+        {% csrf_token %}
+        <button type="submit" class="btn btn-sm btn-primary" name="confirm_event" value="confirm" role="button">Request Participation</button>
```

Comment:
"Request Participation" sounds slightly awkward. Perhaps replace with "Join waitlist".

Reply:
I agree. "Join waitlist" is to the point and consistent to what happens.(The participants join the waitlist after clicking the button)